api/src/Database/DatabaseConnectionFactory.php

```diff
@@ -24,6 +25,16 @@ public function get($databaseType)
             $conn = new \mysqli($host, $isb['user'], $isb['pass'], $dbn, $port);
             $conn->set_charset("utf8mb4");
         }
+        if ($databaseType == 'PureMySQL') {
+            $port = array_key_exists('port', $summarydbconfig) ? $summarydbconfig['port'] : null;
```

Comment:
Can we have a think about what to do if the facility is not running a summary database, without the $summarydbconfig set this code dies. We probably need to carry on and do nothing and make sure the page is fine if we get a null back.